ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -409,6 +409,10 @@ StatusDialog {
                             sourceComponent: CollectiblesSelectionAdaptor {
                                 accountKey: popup.selectedAccount.address
 
+                                networksModel: SortFilterProxyModel {
```

Comment:
Ideally we should take just `networsModel` as a top-level input param and use it as it is, with no extra shenanigans inside. So not in store, but also not here as it shouldn't be this component responsibility to do that. But ofc we can try to improve it later, as it probably triggers more work.

Reply:
hmm I understand, I think first thing missing here is a dedicated networks store. After, I am not sure who would be responsible for transforming this model to suit the needs of the Adaptor here?